app/[lang]/artistry/page.tsx

```diff
@@ -7,15 +7,10 @@ import TitleWithQuote from '~/components/title-with-quote/TitleWithQuote';
 import MusicTableSection from './CompositionTable/MusicTableSelection';
 import { Language } from '~/types/types/language';
 
-import { createRequestContainer } from '~/di/container';
-
 export default async function Artistry({ params }: Readonly<Language>) {
   const { lang } = await params;
-  setRequestLocale(lang);
-  const musicData = await createRequestContainer().resolve('artistryService').getAllCompositions(lang);
-
+  setRequestLocale(lang as 'en' | 'uk');
```

Comment:
```suggestion
  setRequestLocale(lang);
```
`setRequestLocale(lang)` works without it because the lang value comes from the Next Intl routing layer, which guarantees that only 'en' or 'uk' will be passed. So the extra type assertion is not necessary